paper/src/main/java/com/denizenscript/denizen/paper/utilities/PaperAPIToolsImpl.java

```diff
@@ -122,6 +139,17 @@ public void sendSignUpdate(Player player, Location loc, String[] text) {
         player.sendSignChange(loc, components);
     }
 
+    @Override
+    public void sendSignUpdate(Player player, Location loc, String[] text, Side side) {
+        Sign sign = (Sign)loc.getBlock().getState();
+        SignSide signSide = sign.getSide(side);
+        signSide.lines().clear();
+        for (String s : text) {
+            signSide.lines().add(PaperModule.parseFormattedText((s == null ? "" : s), ChatColor.BLACK));
+        }

```

Comment:
The JavaDocs don't seem to specify the list being a live view (I.e. not just a copy or anything), so probably safer to set these like the other methods do